src/data/enat/basic.lean

```diff
@@ -14,6 +14,8 @@ In this file we define `enat` (notation: `ℕ∞`) to be `with_top ℕ` and prov
 about this type.
 -/
 
+example : char_zero (with_top ℕ) := by show_term {apply_instance}
```

Comment:
Causing build failure:

>  Notice: /home/lean/actions-runner/_work/mathlib/mathlib/src/data/enat/basic.lean:17:39:
Try this: exact with_top.char_zero

> Build succeeded, but the following files were noisy:
src/data/enat/basic.lean
Error: Process completed with exit code 1.

bors r-

Reply:
Oops, thanks.